### PR TITLE
doc updates for 2 factor default

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** updated existing docs to reflect upcoming 2 factor default for password factor only uses cases 
- **Is this PR related to a Monolith release?** Yes 2022.04.0

### Resolves:

* [OKTA-477375](https://oktainc.atlassian.net/browse/OKTA-477375)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/create-app-integration.md [hunk 1]
```diff
@@ -14,7 +14,14 @@ Before you integrate Okta authentication into your React app, you need to regist
...
     | Assignments   | Assign users to your app                                |
 
-    > **Note:** Cross-Origin Resource Sharing (CORS) is automatically enabled for the Trusted Origins base URI that you specified in the Admin Console. If you're using the [Okta CLI](https://cli.okta.com/manual/apps/create/) to create your SPA Okta app integration, CORS is also automatically enabled for your base URI. You can verify that both **CORS** and **redirect** are enabled for your app by reviewing the **Security** > **API** > **Trusted Origins** page in the Admin Console.
+5. Select the **Sign On** tab and scroll down to the **User authentication** section. New apps are automatically assigned the shared default [authentication policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-asop). This policy has a catch-all rule that allows a user access to the app using either one or two factors, depending on your org setup.
+6. For this use case, we want to use only the password factor. Click **Edit** and select the “Password only** [preset policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to assign it to your app.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/create-app-integration.md:18]:
```suggestion
6. For this use case, we want to use only the password factor. Click **Edit** and select the **Password only** [preset policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to assign it to your app.
```

File: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/create-app-integration.md [hunk 2]
```diff
@@ -14,7 +14,14 @@ Before you integrate Okta authentication into your React app, you need to regist
...
+
+    > **Note:** Be sure to also [update the password authenticator policy rule](/docs/guides/oie-embedded-common-org-setup/nodejs/main/#update-the-password-authenticator-to-password-only) to not require any additional verification.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-authjs/main/react/create-app-integration.md:20]: same suggestion as above: move note after "Click Save." step?

File: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/angular/create-app-integration.md [hunk 3]
```diff
@@ -14,6 +14,12 @@ The sample application requires an integration with your Okta org to implement a
    * Set **Sign-out redirect URIs** to `http://localhost:8080/`.
 
 1. In the **Assignments** section, select **Allow everyone in your organization to access**.
+1. Click **Save**.
+1. Select the **Sign On** tab and scroll down to the **User authentication** section. New apps are automatically assigned the shared default [authentication policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-asop). This policy has a catch-all rule that allows a user access to the app using either one or two factors, depending on your org setup.
+1. For this use case, we want to use only the password factor. Click **Edit** and select the “Password only** [preset policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to assign it to your app.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/angular/create-app-integration.md:19]:
```suggestion
1. For this use case, we want to use only the password factor. Click **Edit** and select the **Password only** [preset policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to assign it to your app.
```

File: packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/create-app-integration.md [hunk 4]
```diff
@@ -14,7 +14,15 @@ Before integrating Okta authentication into your React app, obtain the OpenID Co
...
     | Assignments   | Assign users for your app.                                |
 
-    > **Note:** Cross-Origin Resource Sharing (CORS) is automatically enabled for the Trusted Origins base URI that you specified in the Admin Console. If you're using the [Okta CLI](https://cli.okta.com/manual/apps/create/) to create your SPA Okta app integration, CORS is also automatically enabled for your base URI. You can verify that both **CORS** and **redirect** are enabled for your app by reviewing the **Security** > **API** > **Trusted Origins** page in the Admin Console.
+5. Select the **Sign On** tab and scroll down to the **User authentication** section. New apps are automatically assigned the shared default [authentication policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-asop). This policy has a catch-all rule that allows a user access to the app using either one or two factors, depending on your org setup.
+
+6. For this use case, we want to use only the password factor. Click **Edit** and select the “Password only** [preset policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to assign it to your app.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/create-app-integration.md:19]:
```suggestion
6. For this use case, we want to use only the password factor. Click **Edit** and select the **Password only** [preset policy](https://help.okta.com/okta_help.htm?type=oie&id=ext-preset-auth-policies) to assign it to your app.
```

**Review by danielmaharry-okta — APPROVED** [review]
LGTM!

**Pull request merged**